packages/dds/tree/src/test/simple-tree/benchmarkUtilities.ts

```diff
@@ -0,0 +1,150 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { SchemaFactory } from "../../simple-tree/index.js";
+import { hydrate } from "./utils.js";
+
+const schemaFactory = new SchemaFactory("test");
+
+/**
+ * A recursive tree node with a single leaf value
+ */
+export class DeepTreeNode extends schemaFactory.objectRecursive("deep-tree-node", {
+	node: schemaFactory.optionalRecursive([() => DeepTreeNode]),
+	value: schemaFactory.optional(schemaFactory.number),
+}) {}
+
+/**
+ * A wide tree node with all leaves having the same value
+ */
+export class WideTreeNode extends schemaFactory.object("wide-tree-node", {
+	values: schemaFactory.array(schemaFactory.number),
+}) {}
```

Comment:
This is adding an unneeded layer to the tree (an object node wrapping an array node (which wraps a sequence field)).

```suggestion
export class WideTreeNode extends schemaFactory.array("wide-tree-node", schemaFactory.number) {}
```
Is a simpler way to do this without the extra layer in the tree.

If you prefer to use a structurally named array, you could also use:

```suggestion
export const WideTreeNode = schemaFactory.array(schemaFactory.number);
export type WideTreeNode = NodeFromSchema<typeof WideTreeNode>;
```

Reply:
Updated